x/csr/types/msg.go

```diff
@@ -87,43 +95,109 @@ func (msg *MsgRegisterCSR) GetSigners() []sdk.AccAddress {
 	return []sdk.AccAddress{sdk.MustAccAddressFromBech32(msg.Deployer)}
 }
 
+// method to create a new MsgUpdateCSR message, deployer is the sdk.AccAddress of the deployer of the CSR being updated
+// pooladdr is the sdk.AccAddress of the CSR pool being updated
+func NewMsgUpdateCSR(
+	deployer,
```

Comment:
```suggestion
	deployer sdk.AccAddress,
```

Reply:
deployer and poolAddress are of the same type, so the first type assertion (line 101) can be ignored